2024-10-25 14:49:00,705 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 14:49:13,353 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 14:49:17,017 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:49:17,018 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:49:17,568 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 14:49:18,200 ERROR: An error occurred in index route: Could not build url for endpoint 'barcode.fetch_barcode'. Did you mean 'pnp.fetch_barcode' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:117]
2024-10-25 14:49:54,664 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:49:54,665 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:49:55,293 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 14:50:49,958 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 14:50:51,522 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:50:51,523 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:50:52,141 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 14:50:52,549 ERROR: An error occurred in index route: Could not build url for endpoint 'barcode.fetch_barcode'. Did you mean 'pnp.fetch_barcode' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:124]
2024-10-25 14:50:54,082 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:50:54,082 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:50:54,917 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 14:50:55,888 ERROR: An error occurred in index route: Could not build url for endpoint 'barcode.fetch_barcode'. Did you mean 'pnp.fetch_barcode' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:124]
2024-10-25 14:51:08,105 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 14:51:09,218 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:51:09,220 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:51:09,911 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 14:51:10,497 ERROR: An error occurred in index route: Could not build url for endpoint 'barcode.fetch_barcode'. Did you mean 'barcode.handle_action' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:124]
2024-10-25 14:51:14,635 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:51:14,636 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:51:15,191 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 14:51:15,585 ERROR: An error occurred in index route: Could not build url for endpoint 'barcode.fetch_barcode'. Did you mean 'barcode.handle_action' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:124]
2024-10-25 14:51:30,893 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 14:51:33,354 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:51:33,355 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:51:33,357 INFO: Reset processing flags for SKU 'DB-0064' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 14:51:33,965 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 14:51:33,965 INFO: Stale processing flags have been reset. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:123]
2024-10-25 14:51:34,599 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 14:51:36,645 INFO: Fetching details for product_code: 000000000000852854_EA [in /home/creep/projects/validate-barcode/app/routes/barcode.py:184]
2024-10-25 14:51:36,649 INFO: Fetching details for product_code: 000000000000852854_EA [in /home/creep/projects/validate-barcode/app/routes/barcode.py:184]
2024-10-25 14:51:36,420 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:51:36,421 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:51:36,431 INFO: Retail price for SKU 000000000000852854_EA: 0.0 [in /home/creep/projects/validate-barcode/app/routes/barcode.py:235]
2024-10-25 14:51:36,796 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 14:51:36,797 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 14:51:36,799 INFO: Retail price for SKU 000000000000852854_EA: 0.0 [in /home/creep/projects/validate-barcode/app/routes/barcode.py:235]
2024-10-25 15:06:16,236 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 15:06:16,618 INFO: Inventory Validation Startup [in /home/creep/projects/validate-barcode/app/__init__.py:29]
2024-10-25 15:06:21,609 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 15:06:21,613 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 15:06:21,617 INFO: Reset processing flags for SKU 'DE-0365' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,617 INFO: Reset processing flags for SKU 'DI-0003' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,618 INFO: Reset processing flags for SKU 'DB-0167' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,618 INFO: Reset processing flags for SKU 'DE-0315' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,618 INFO: Reset processing flags for SKU 'DA-0764' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,618 INFO: Reset processing flags for SKU 'DA-0048' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,618 INFO: Reset processing flags for SKU 'DA-0069' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,619 INFO: Reset processing flags for SKU 'DB-0040' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,619 INFO: Reset processing flags for SKU 'DA-0362' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,619 INFO: Reset processing flags for SKU 'DC-0217' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,619 INFO: Reset processing flags for SKU 'DJ-0042' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:21,619 INFO: Reset processing flags for SKU 'DA-0645' due to timeout. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:119]
2024-10-25 15:06:22,269 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 15:06:22,270 INFO: Stale processing flags have been reset. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:123]
2024-10-25 15:06:23,076 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 15:06:24,229 ERROR: An error occurred in index route: Could not build url for endpoint 'barcode.fetch_barcode'. Did you mean 'pnp.fetch_barcode' instead? [in /home/creep/projects/validate-barcode/app/routes/main.py:110]
2024-10-25 15:06:55,328 INFO: Loaded main_inventory_staging.xlsx from S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:32]
2024-10-25 15:06:55,337 INFO: 'processing_timestamp' column converted to datetime. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:82]
2024-10-25 15:06:55,970 INFO: Saved main_inventory_staging.xlsx to S3. [in /home/creep/projects/validate-barcode/app/utils/inventory.py:101]
2024-10-25 15:06:58,313 INFO: Fetching details for product_code: 000000000000145230_EA [in /home/creep/projects/validate-barcode/app/routes/search_routes/pnp_search.py:76]
